web/app/js/components/Navigation.jsx

```diff
@@ -402,7 +403,7 @@ class NavigationBase extends React.Component {
   }
 
   menuItem(path, title, icon, onClick) {
```

Comment:
Why isn't this a component? It seems like a perfect opportunity to pull this code out and simplify things.